tests/Integration/Behaviour/Features/Context/Domain/AttributeGroupFeatureContext.php

```diff
@@ -94,7 +94,7 @@ public function assertAttributeGroupIsDeleted(string $reference): void
         try {
             $this->getQueryBus()->handle(new GetAttributeGroupForEditing($attributeGroupId));
 
-            throw new NoExceptionAlthoughExpectedException(sprintf('Attribute group %s exists, but it was expected to be deleted', $reference));
+            throw new NoExceptionAlthoughExpectedException(sprintf('Attribute %s exists, but it was expected to be deleted', $reference));
```

Comment:
here as well Attribute group was correct (as it is just exception message and we must be precise and techical here. And technical term is AttributeGroup. Even the query is named that way)